### DISCUSSION TITLE
Create a 3 level trigger?

### DISCUSSION THREAD

**Discussion by StephenMSmith** [Rem0o/FanControl.Releases#2730] [Q&A] [answered]
Is it possible to create a trigger with 3 levels?  I currently use a trigger for my CPU and Rear fan b/c there are various RPMs that create an annoying sound, so I use a trigger for 35% @ 50c and 100% @ 70c.  This works but I would love to have a step in the middle.

**Comment by Rem0o** [comment, marked as answer]
Create 2 triggers, create a mix, add the two triggers to the mix.

**Reply by StephenMSmith** [reply]
Oh wow, works like a charm.  Thanks for this awesome app!
